src/definitions/wrapping.ts

```diff
@@ -164,61 +164,32 @@ export function isNexusPlugin(obj: any): obj is NexusPlugin {
   return isNexusStruct(obj) && obj[NexusWrappedSymbol] === NexusTypes.Plugin
 }
 
-type NexusWrapKind = { type: 'List'; nullable: boolean } | { type: 'NamedType'; nullable: boolean }
+export type NexusWrapKind = 'NonNull' | 'Null' | 'List' | 'WrappedDef'
```

Comment:
What is `WrappedDef` for?

Reply:
list('String') -> ['WrappedDef', 'List]. We need the WrappedDef to be able to apply the nonNull() if nonNullDefault is true

Reply:
I see, makes sense.